content/webapp/components/IIIFViewer/IIIFViewerImage.tsx

```diff
@@ -60,8 +62,24 @@ const IIIFViewerImage = (
           clickHandler && clickHandler();
         }
       }}
-      onError={() => {
-        errorHandler && errorHandler();
+      onError={({ currentTarget }) => {
+        // Hack/workaround
+        // If the image fails to load it may be because of a size limit,
+        // see: https://wellcome.slack.com/archives/CBT40CMKQ/p1691050149722109,
+        // so first off we try a smaller image
+        if (tryLoadingSmallerImg) {
+          setTryLoadingSmallerImg(false); // prevent looping if image fails to load again
+          const isPortrait = Boolean(height && height > width);
+          const newSrc = isPortrait
+            ? convertIiifImageUri(currentTarget.src, 1000, true)
```

Comment:
we should use the maxWidth property of the image, e.g. https://iiif.wellcomecollection.org/image/b18023010_0001.JP2/info.json to set sizes.  We could check for this before requesting images in the first place, but not sure it's a good idea to do this for everything, when it is only a relatively small subset of works that have this problem